app/js/components/rule/ruleGroupCard/ruleGroupCard.jade

```diff
@@ -22,7 +22,7 @@
       i.fa.fa-arrow-right
     card
       card-header
-        .md-card-header-title(translate) {{index + 1}} of {{plugin.rules.length}}: {{rule.description}}
+        .md-card-header-title(translate) {{index + 1}} of {{:: plugin.rules.length}}: {{rule.description}}
```

Comment:
plugin.rules.length cannot be static - the length may change e.g. when rules with no hits are unhidden with the button

Reply:
Wow! Sharp! Thanks man!